### ISSUE TITLE
[SF] [A11y] [ui5-avatar] Avatar is announced as "clickable graphic" even when not interactive

### ISSUE THREAD

**Issue by Neeeko** [SAP/ui5-webcomponents#3683]
### **Bug Description**
The ui5-avatar component is announced by screen readers as "clickable graphic" even when the mode is not interactive.

![Screenshot 2021-08-17 at 14 44 47](https://user-images.githubusercontent.com/7316793/129729218-c5ce2044-f754-4715-a712-21888b877327.png)

### **Expected Behavior**
When the ui5-avatar is _not_ interactive (focusable/clickable), it shouldn't be announced as "clickable". Only the accessible name should be announced by screen readers.

### **Isolated Example**
https://codesandbox.io/s/ui5-webcomponents-forked-yftf5?file=/index.html

### **Context**
- UI5 Web Components version: 0.31.9
- OS/Platform: Windows 10 (using NVDA as screen reader)
- Browser: Google Chrome
- Affected component: ui5-avatar

### **Priority**
- [ ] Low
- [x] Medium
- [ ] High
- [ ] Very High

### **Stakeholder Info** (if applicable)

- Organization: SuccessFactors

**Comment by unazko** [comment]
Hello @SAP/ui5-webcomponents-topic-p,

The issue is reproducible on all available ui5-webcomponents version via the provided steps and NVDA screen reader.
Also the issue isn't reproducible by using jaws 2019, jaws 2021 or Windows Narrator as they announce:
"My Avatar graphic".

Not sure, which screen readers are fully supported.

Best Regards,
Boyan Rakilovski

**Comment by Neeeko** [comment]
Hi,
Thanks for addressing this issue!
If possible, we would appreciate if this fix could be merged and deployed as soon as possible as we have our SF code complete next week.
Thanks!

**Comment by fifoosid** [comment]
Hi @Neeeko 

The fix will be available in the next release of the 0.31 branch
